conf/cassandra.yaml

```diff
@@ -359,9 +359,9 @@ prepared_statements_cache_size_mb:
 # NOTE: if you reduce the size, you may not get you hottest keys loaded on startup.
 #
 # Default value is empty to make it "auto" (min(5% of Heap (in MB), 100MB)). Set to 0 to disable key cache.
-key_cache_size_in_mb:
+key_cache_size:
 
-# Duration in seconds after which Cassandra should
+# DurationSpec in seconds after which Cassandra should
```

Comment:
Looks like a few instances in the YAML where we accidentally changed this to `DurationSpec`.

Reply:
Corrected all of them, thanks!